app/models/site_update.rb

```diff
@@ -0,0 +1,77 @@
+class SiteUpdate < ApplicationRecord
+  include IsParamaterisedSTI
+  include Mandate
+
+  self.class_suffix = :update
+  self.i18n_category = :site_updates
+
+  scope :published, -> { where('published_at < ?', Time.current) }
+  scope :for_track, ->(track) { where(track: track) }
+  scope :sorted, -> { order(published_at: :desc) }
+
+  belongs_to :author, optional: true, class_name: "User"
+  belongs_to :pull_request, optional: true, class_name: "Github::PullRequest"
+
+  before_validation only: :create do
+    self.published_at = Time.current + 3.hours unless published_at
```

Comment:
The default is for an update to be published 3 hours later?

Reply:
Yeah, so my thinking here is that we give maintainers 3 hours to add a fuller version of the post before it gets published. You think that's a sensible timeframe?

Reply:
(They can still do it afterwards, but this gives them space to get the announcement right when the first people see it)

Reply:
I think that's reasonable. Publishing would imply that it's "done" anyways.